exporters/InventorRobotExporter/README.md

```diff
@@ -1,20 +1,26 @@
-# InventorRobotExporter
+# InventorRobotExporter (Visual Studio Solution)
```

Comment:
Is it necessary to say `(Visual Studio Solution)`? If so, I would also add it to the FusionRobotExporter README title.

Reply:
Okay, removed